docs/docs/building-with-components.md

```diff
@@ -118,7 +118,7 @@ export const pageQuery = graphql`
 `src/html.jsx` is responsible for everything other than where Gatsby lives in
 the `<body />`.
 
-In this file you can modify the `<head>` metadata, general structure of the
+In this file, you can modify the `<head>` metadata and the general structure of the
```

Comment:
To reduce the number of "the"'s in this sentence, you can delete "the" from before the word "general". It would then read: "modify the `head` metadata and general structure of...", so the "modify the" applies to both direct objects. I believe the way you've done it is also correct, so just thought you might be curious about other ideas since it seems that your grammar skills are top notch and maybe you like learning that kind of stuff!